tests/trnsysGUI/pythonInterface/regimeExporter/testHydraulicModesGui.py

```diff
@@ -0,0 +1,123 @@
+import pathlib as _pl
```

Comment:
are the pdfs created when running this?
I suggest turning that off for this level of the tests.
Mock the function _makeDiagrams.